tests/integration/testlib/ovslib.py

```diff
@@ -84,6 +87,18 @@ def _get_port(self, name):
             (port for port in ports if port[OVSBridge.Port.NAME] == name), None
         )
 
+    def del_port(self, name):
+        new_ports = [
```

Comment:
Can you use here `filter`?

Also, for readability (reduce the line breaking), could you please place `self._bridge_iface[OVSBridge.CONFIG_SUBTREE][OVSBridge.PORT_SUBTREE]` into a temp variable?